helpers/calendar/all_analytics_helper.py

```diff
@@ -67,43 +63,40 @@ def get_all_analytics(self, query, start_date, end_date, location_id, unconverte
         rooms = query.filter_by(state="active", location_id=location_id).all()
         room_analytics = []
         bookings = 0
+        total_checkins = 0
+        bookings_count = []
+        all_events = []
         for room in rooms:
-            cancellations = 0
             checkins = 0
             events = []
+            duaration_in_minutes = 0
             try:
                 events_result = CommonAnalytics.get_all_events_in_a_room(
                     self, room.id, start_date, end_date)
             except GraphQLError:
                 continue
-            bookings_count = AllAnalyticsHelper.bookings_count(self, unconverted_dates, events_result) # noqa
+            all_events += events_result
             bookings += len(events_result)
             for event in events_result:
                 if event.checked_in:
                     checkins += 1
-                if event.cancelled:
-                    cancellations += 1
-                duaration_in_minutes = CommonAnalytics.get_time_duration_for_event(self, event.start_time, event.end_time) # noqa
-                current_event = Event(
-                    duration_in_minutes=duaration_in_minutes)
-                events.append(current_event)
+                duaration_in_minutes += CommonAnalytics.get_time_duration_for_event(self, event.start_time, event.end_time) # noqa
+            current_event = Event(
+                duration_in_minutes=duaration_in_minutes)
+            events.append(current_event)
             room_analytic = {
                 'number_of_meetings': len(events_result),
                 'room_name': room.name,
-                'cancellations': cancellations,
-                'checkins': checkins,
-                'cancellations_percentage': percentage_formater(cancellations, bookings), # noqa
-                'checkins_percentage': percentage_formater(checkins, bookings),
-                # TODO - work on app_bookings and autocancellation logic
-                'app_bookings': cancellations,  # Place holder
-                'app_bookings_percentage': percentage_formater(cancellations, bookings), # noqa
-                'percentage_share': percentage_formater(
-                    len(events_result), bookings
-                ),
+                'percentage_share': len(events_result),
                 'room_events': events,
-                'bookings_count': bookings_count,
-                'auto_cancellations': 0  # Placeholder
             }
             room_analytics.append(room_analytic)
             room_analytics.sort(key=lambda x: x['number_of_meetings'], reverse=True) # noqa
-        return room_analytics, bookings
+            total_checkins += checkins
+        percentages_dict = {
+            'total_checkins': total_checkins,
+            'total_auto_cancellations': 0, # This only a placeholder, logic is yet to be implemented
```

Comment:
Implement this logic as the fields in question have been added to the event object